_posts/2025-02-05-#31363.md

```diff
@@ -10,17 +10,263 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### General Cluster Mempool
+
+Skip this section if you are just interested in reviewing the code and don't need to hear the pitch for why.
+
+The primary goal of [Cluster Mempool](https://github.com/bitcoin/bitcoin/issues/30289) is to enable the mempool to more
```

Comment:
Probably worth linking to the optech write-up here too, it's good! https://bitcoinops.org/en/topics/cluster-mempool/